exporter.go

```diff
@@ -263,11 +299,28 @@ func (b *Exporter) Listen(e <-chan Events) {
 				eventStats.WithLabelValues("gauge").Inc()
 
 			case *TimerEvent:
-				summary := b.Summaries.Get(
-					b.suffix(metricName, "timer"),
-					prometheusLabels,
-				)
-				summary.Observe(event.Value())
+				timerType := ""
+				if mapping != nil {
+					timerType = mapping.TimerType
+				}
+				if timerType == "" {
+					timerType = b.mapper.Defaults.TimerType
+				}
+
+				if timerType == "histogram" {
+					histogram := b.Histograms.Get(
+						b.suffix(metricName, "timer"),
```

Comment:
Hmm, I think we should only suffix metrics that don't have an explicitly mapped metric name (if there is a mapping, use exactly the metric name provided in the mapping). But I see we also do that for gauges and counters already. So that should probably be fixed in a separate change.